.github/workflows/protocol.yml

```diff
@@ -66,21 +66,12 @@ jobs:
         working-directory: ./packages/protocol
         run: pnpm compile:l1 && pnpm snapshot:l1 && pnpm layout:l1
 
-      - name: Check for changes
```

Comment:
I tried these modifications in the protocol.yml file but it doesn't seem to work.

Reply:
These changes should be reverted!